config/packages/framework.yaml

```diff
@@ -1,17 +1,25 @@
+# see https://symfony.com/doc/current/reference/configuration/framework.html
 framework:
     secret: '%env(APP_SECRET)%'
     #csrf_protection: true
-    #http_method_override: true
+    #http_method_override: false
```

Comment:
this is different to sulu but we actually enable http_method_override in our `public/index.php`